vk-components/package/_scss/_vk-post-type-card.scss

```
@@ -273,7 +273,7 @@ https://github.com/vektor-inc/vektor-wp-libraries
 		}
 	}
 	.card-body {
-		height: 100%;
+		height: auto;
```

@mtdkei ありがとうございます。
これ 100% に対して padding 分を加算されてしまって下にはみ出してる状態なので、

```suggestion
		height: 100%;
		box-sizing:border-box;
```

で処理した方が自然じゃない（・ｗ・？

---

@kurudrive 
確かにそうでした、うっかりしてました。
確認して修正しました。